### PR TITLE
Add parent method to Gollum::Git::Commit

### PR DESCRIPTION
**Pull request by heavywatal**
Related to gollum/gollum#1543

### CODE REVIEW

File: spec/gollum_git_commit_spec.rb [hunk 1]
```diff
@@ -38,4 +39,7 @@
     expect(commit.tree).to be_a Gollum::Git::Tree
   end
 
+  it "returns a Gollum::Git::Commit object for parent" do
+    expect(commit.parent).to be_a Gollum::Git::Commit
```
Comment (bartkamphorst) [inline spec/gollum_git_commit_spec.rb:43]: What if there is no parent?

Comment (dometto) [reply spec/gollum_git_commit_spec.rb:43]: Then it should return `nil`, of course :) But how could we test this in the specs?

Comment (heavywatal) [reply spec/gollum_git_commit_spec.rb:43]: Something like `expect(repo.commits.first.parent).to eq nil`?

Comment (dometto) [reply spec/gollum_git_commit_spec.rb:43]: Yes, but you'll either have to use mocking to get the `Rugged::Commit`'s parents to be empty, or find a commit for which that's the case in the test repository

Comment (heavywatal) [reply spec/gollum_git_commit_spec.rb:43]: Sorry I don't get the point. I thought `repo.commits.first` always returns the root commit with no parent.

Comment (dometto) [reply spec/gollum_git_commit_spec.rb:43]: eh, I guess that's right. Ignore me 😆

**Pull request merged**